implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -72,13 +72,13 @@ export default () => {
 
 ### Custom search URL parameters mapping
 
-Configuration accepts optional array of URL parameter mapping functions, which allow to change behaviour of any search parameter. `customUrlMappings` accepts objects which must contain `encode` and `decode` functions.
+Manual URL Parameter Mapping (Deprecated)
```

Comment:
I am ok with both hiding and deprecating the old functionality
is it deprecated in the typedoc documentation or hidden?